drivers/sensor/Kconfig

```diff
@@ -254,4 +254,6 @@ source "drivers/sensor/ite_vcmp_it8xxx2/Kconfig"
 
 source "drivers/sensor/pcnt_esp32/Kconfig"
 
+source "drivers/sensor/icp10125/Kconfig"
```

Comment:
Please move this up to sort in alpha order (yes there are a few outliers at the end that need to be fixed in another PR)

Reply:
fixed it.